src/builders.ts

```diff
@@ -40,9 +41,10 @@ export function _buildTodoDatum(
   lintResult: LintResult,
   lintMessage: LintMessage
 ): TodoData {
+  const filePath = isAbsolute(lintResult.filePath) ? relative(baseDir, lintResult.filePath) : lintResult.filePath;
   return {
     engine: getEngine(lintResult),
-    filePath: posix.relative(baseDir, lintResult.filePath),
+    filePath: slash(filePath),
```

Comment:
Can we add a comment here to remove this once the fix has been pushed to node?

Reply:
Done.